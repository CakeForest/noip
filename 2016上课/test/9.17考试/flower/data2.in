10 6
1 2 5
2 3 4
1 0 8
2 2 5
1 4 4
1 2 3
