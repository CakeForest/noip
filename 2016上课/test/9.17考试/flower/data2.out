2 6
2
0 9
4
4 5
2 3
